collect_app/src/main/java/org/odk/collect/android/external/ExternalSQLiteOpenHelper.java

```diff
@@ -263,4 +265,16 @@ private void onProgress(String message) {
             formLoaderTask.publishExternalDataLoadingProgress(message);
         }
     }
+
+    /**
+     * replaceByteOrderMark
+     *
+     * Removes the presence of a Byte Order Mark from the initial UTF-8 String
+     *
+     * @param firstLine String to scan for a Byte Order Marker
+     * @return String with any Byte Order Marker removed
+     */
+    private String replaceByteOrderMark(String firstLine) {
```

Comment:
Could you just change this method name? I think `removeByteOrderMark` would be better since it's removing not replacing.

Reply:
No problem, agreed